docs/getting-started.md

```diff
@@ -60,6 +60,12 @@ Before you begin, ensure you have the following software installed on your machi
      npm run pkg @cloudoperators/juno-app-example
      ```
 
+   - **Generator:Package**
+     The npm run generate:package command is used to generate a new package in a project. When running this command, you need to provide a name for the package that starts with `@cloudoperators/juno-`.
```

Comment:
Would be a good idea to describe here what is a package and what would be the use case?